README.md

```diff
@@ -36,43 +36,48 @@ Scoop is a command-line installer for Windows.
+Scoop installs apps from the command line with a minimal amount of friction. It:
 
-- Eliminates permission popup windows
-- Hides GUI wizard-style installers
-- Prevents PATH pollution from installing lots of programs
-- Avoids unexpected side-effects from installing and uninstalling programs
-- Finds and installs dependencies automatically
-- Performs all the extra setup steps itself to get a working program
+- Eliminates [User Account Control](https://learn.microsoft.com/windows/security/application-security/application-control/user-account-control/) (UAC) prompt notifications.
+- Hides the graphical user interface (GUI) of wizard-style installers.
+- Prevents polluting the `PATH` environment variable. Normally, this variable gets cluttered as different apps are installed on the device.
+- Avoids unexpected side effects from installing and uninstalling apps.
+- Resolves and installs dependencies automatically.
+- Performs all the necessary steps to get an app to a working state.
 
-Scoop is very scriptable, so you can run repeatable setups to get your environment just the way you like, e.g.:
+Scoop is quite script-friendly. Your environment can become the way you like by using repeatable setups. For example:
 
-```powershell
+```console
 scoop install sudo
 sudo scoop install 7zip git openssh --global
 scoop install aria2 curl grep sed less touch
 scoop install python ruby go perl
 ```
 
-If you've built software that you'd like others to use, Scoop is an alternative to building an installer (e.g. MSI or InnoSetup) — you just need to zip your program and provide a JSON manifest that describes how to install it.
+If you have built software that you would like others to use, Scoop is an alternative to building an installer (like MSI or InnoSetup). You just need to compress your app to a `.zip` file and provide a JSON manifest that describes how to install it.
 
 ## Installation
 
-Run the following command from a **non-admin** PowerShell to install scoop to its default location `C:\Users\<YOUR USERNAME>\scoop`.
+Run the following commands from a regular (non-admin) PowerShell terminal to install Scoop:
 
 ```powershell
-iwr -useb get.scoop.sh | iex
+Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
+Invoke-RestMethod -Uri https://get.scoop.sh | Invoke-Expression
 ```
 
-Advanced installation instruction and full documentation of the installer are available in [ScoopInstaller/Install](https://github.com/ScoopInstaller/Install). Please create new issues there if you have questions about the installation.
+**Note**: The first command makes your device allow running the installation and management scripts. This is necessary because Windows 10 client devices restrict execution of any PowerShell scripts by default.
 
-## [Documentation](https://github.com/ScoopInstaller/Scoop/wiki)
+It will install Scoop to its default location:
+
+`C:\Users\<YOUR USERNAME>\scoop`
```

Comment:
This could be
`C:\Users\%USERNAME%\scoop`
or better yet:
`%USERPROFILE%\scoop`
?

Reply:
I think `C:\Users\<YOUR USERNAME>\scoop` is more user-friendly, as people might not know env var expansion. 

I feel a Readme should be as user-friendly as possible.